Compile errors

Thereafter bootstrapping with ./autogen.sh

./configure returns error:
```./config.status: line 1299: `$as_dirname -- "$_am_arg" ||
$as_expr X"$_am_arg" : 'X\(.*[^/]\)//*[^/][^/]*/*$' \| 	 X"$_am_arg" : 'X\(//\)[^/]' \| 	 X"$_am_arg" : 'X\(//\)$' \| 	 X"$_am_arg" : 'X\(/\)' \| . 2>/dev/null ||
$as_echo X"$_am_arg" |
    sed '/^X\(.*[^/]\)\/\/*[^/][^/]*\/*$/{
	    s//\1/
	    q
	  }
	  /^X\(\/\/\)[^/].*/{
	    s//\1/
	    q
	  }
	  /^X\(\/\/\)$/{
	    s//\1/
	    q
	  }
	  /^X\(\/\).*/{
	    s//\1/
	    q
	  }
	  s/.*/./; q'`/stamp-h1: Permission denied```
But works well if executed with sudo.

Then make install returns error:
```make: *** No rule to make target 'ttsnake.sh', needed by 'ttsnake'.  Stop.```

it compiles ttsnack.bin, but don't ttsnake neither installs.

--- a/src/ttsnake.sh
+++ b/src/ttsnake.sh
@@ -1,0 +1,18 @@
+#!/bin/sh
+
+ARG="$0"
+PROGRAM="$(dirname $ARG)/ttsnake.bin"
+
+
+
+if [ -f "$PROGRAM" ]; then
+    if [ -z "$1" ] ; then
+	./$PROGRAM -d scenes
+    else
+	./$PROGRAM "$1"
+    fi
+else
+    echo "Program not found."
+    exit 1
+fi
+       

